FlameBot.js

```diff
@@ -38,20 +38,22 @@ class FlameBot {
      * @type {Object}
      */
     this.telegram = telegram;
+
+    /**
+     * The bots username as a Promise
+     * @type {Promise.<string>}
+     */
+    this.usernamePromise = new Promise((resolve, reject) => {
+      telegram.getMe().then((me) => {
+        resolve(me.username);
+      }, reject);
```

Comment:
That would work if I would pass on the inner Promise (`resolve(telegram.getMe())`) as `resolve` “follows” Promises (more exactly, it follows objects with a `then` method). But in this situation, how would the outer Promise know about the inner Promise?